src/test/java/org/cactoos/list/JoinedTest.java

```diff
@@ -435,4 +439,25 @@ public void itemAndList() {
             )
         ).affirm();
     }
+
+    @Test
+    public void infersCorrectly() {
+        new Assertion<>(
+            "must be able to infer type of elements",
```

Comment:
@andreoss while we are at it, let's put an upper case at the beginning of the sentence

Reply:
@victornoel Done